test/acceptance/behave/components/ngsiv2/entities/delete_entities.feature

```diff
@@ -0,0 +1,402 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# author: 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed
+#        -tg=-skip
+#
+
+Feature: delete entities requests (POST) using NGSI v2 API. "DELETE" - /v2/entities/
+  As a context broker user
+  I would like to delete entities requests using NGSI v2 API
+  So that I can manage and use them in my scripts
+
+  Actions Before the Feature:
+  Setup: update properties test file from "epg_contextBroker.txt" and sudo local "false"
+  Setup: update contextBroker config file
+  Setup: start ContextBroker
+  Check: verify contextBroker is installed successfully
+  Check: verify mongo is installed successfully
+
+  Actions After each Scenario:
+  Setup: delete database in mongo
+
+  Actions After the Feature:
+  Setup: stop ContextBroker
+
+  @happy_path
+  Scenario:  delete several entities using NGSI v2 API
```

Comment:
Try to start the scenario title with capital case and with a tittle more specific that correctly identifies the test and shows how it differs with the rest. This could help in the execution report just showing the tittles. 

just a proposal from:
delete several entities using NGSI v2 API
delete entities by ID using NGSI v2 with several service header values
to:
Entities deletion using NGSI v2 API
Entities deletion using NGSI v2 API searching by ID with extra headers